.builds/openbsd.yml

```diff
@@ -22,7 +22,5 @@ tasks:
       GOARCH=386 go build ./...
       go test -c .
       doas bash -c 'WGCTRL_INTEGRATION=yesreallydoit ./wgctrl.test -test.v -test.run TestIntegration'
-      # TODO: re-enable once Go 1.19 is available in openbsd/latest and wireguard-go can be built
-      exit 0
```

Comment:
Is this change necessary? Seems unrelated to the purpose of the PR?

Reply:
I dont remember that I've committed these changes, @mdlayher might have the answer